controllers/articles.controllers.js

```diff
@@ -0,0 +1,12 @@
+const { selectArticlesById } = require('../models/articles.models');
+
+exports.getArticlesById = (req, res, next) => {
+	const { article_id } = req.params;
+	selectArticlesById(article_id)
```

Comment:
When selecting article by ID, it's always gonna be just one article at a time. Please replace all the words 'articles' with 'article.